### ISSUE TITLE
Error in Knowledgebase plugin

### ISSUE THREAD

**Issue by vikasprogrammer** [freescout-helpdesk/freescout#1070]
When I click on new Article in KB plugin, I get the following error in Laravel logs

```
[2021-02-25 13:11:24] production.ERROR: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'mailbox_id' in 'where clause' (SQL: select * from `kb_articles` where `mailbox_id` = 9) 
```

**Comment by freescout-helpdesk** [comment]
Can you provide SSH and web access to your FreeScout? support@freescout.net

**Comment by vikasprogrammer** [comment]
done!

**Comment by freescout-helpdesk** [comment]
The issue has been fixed in the Knowledge Base Module v1.0.5

If your Modules page is working, update the module to v1.0.5 and reactivate it. 

Otherwise:
1) Update FreeScout to the latest version (1.6.16): https://github.com/freescout-helpdesk/freescout/wiki/Updating-FreeScout
2) Rename /Modules/KnowledgeBase into /Modules/KnowledgeBase2
3) Open https://your-freescout.com/modules/list two times, now KnowledgeBase module will be deactivated.
4) Rename /Modules/KnowledgeBase2 back into /Modules/KnowledgeBase
5) Now update the module to the latest version and activate it.

**Comment by diguinho2018** [comment]
> 
> 
> The issue has been fixed in the Knowledge Base Module v1.0.5
> 
> If your Modules page is working, update the module to v1.0.5 and reactivate it.
> 
> Otherwise:
> 
>     1. Update FreeScout to the latest version (1.6.16): https://github.com/freescout-helpdesk/freescout/wiki/Updating-FreeScout
> 
>     2. Rename /Modules/KnowledgeBase into /Modules/KnowledgeBase2
> 
>     3. Open https://your-freescout.com/modules/list two times, now KnowledgeBase module will be deactivated.
> 
>     4. Rename /Modules/KnowledgeBase2 back into /Modules/KnowledgeBase
> 
>     5. Now update the module to the latest version and activate it.

Just took note of these steps and will do this if something goes wrong.
Maybe you could add this steps to Wiki to be always on radar?
